src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -350,9 +350,9 @@ class CrossDeviceLinkUI extends Component {
                   style[view.className]
                 )}
                 ref={(node) => (this.viewOptionBtn = node)}
-                onClick={() =>
-                  preventDefaultOnClick(this.handleViewOptionSelect(view.id))
-                }
+                onClick={preventDefaultOnClick(() =>
```

Comment:
Out of scope: IMO we'll have troubles in the future with this helper when converting the related files into TypeScript. So far no other places (which consume this helper) were converted to TypeScript so the risks are still unknown.
Btw I prefer to make it explicit: just call `event.preventDefault()` in-place, we probably don't need this helper.

Reply:
Yeah, I agree. I find it's clearer when `event.preventDefault()` is called in place & we don't have to deal with passing the next/actual function at onClick event on to the helper as a callback correctly. Which is the mistake I made here. 🙈